GuidedTutorials/MultiFab/main.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
```

Comment:
Checkout these sections how to support CUDA and HIP with CUPY;
- https://github.com/ECP-WarpX/impactx/blob/23.08/examples/fodo/run_fodo_programmable.py#L69-L80
- https://docs.cupy.dev/en/stable/user_guide/basic.html#how-to-write-cpu-gpu-agnostic-code